releases/v1.6.7.toml

```diff
@@ -0,0 +1,26 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.6.6"
+
+pre_release = false
+
+preface = """\
+The seventh patch release for containerd 1.6 contains various fixes,
+includes a new version of runc and adds support for ppc64le and riscv64 builds.
+
+### Notable Updates
+
+* **Update runc to v1.1.3** ([#7036](https://github.com/containerd/containerd/pull/7036))
+* **Seccomp: Allow clock_settime64 with CAP_SYS_TIME** ([#7172](https://github.com/containerd/containerd/pull/7172))
+* **Fix WWW-Authenticate parsing** ([#7131](https://github.com/containerd/containerd/pull/7131))
+* **Support RISC-V 64 and ppc64 builds** ([#7170](https://github.com/containerd/containerd/pull/7170))
```

Comment:
```suggestion
* **Support RISC-V 64 and ppc64le builds** ([#7170](https://github.com/containerd/containerd/pull/7170))
```

Reply:
It should be also noted that riscv64 needs runc 1.2 (unreleased)

Reply:
Updated. Let me know if the runc wording can be more specific